lib/presentation/search_page/search_page.dart

```diff
@@ -19,21 +38,36 @@ class SearchPage extends StatelessWidget {
         elevation: 0.0,
         backgroundColor: Colors.white,
         actions: [
+
+          IconButton(
+            onPressed: () {
+              context.read<AuthBloc>().add(SignOutRequestEvent());
+              context.read<UserCubit>().initialUser();
```

Comment:
Maybe add method .initalUser to AuthBloc when signed out? and maybe change the name method to setUserAsInitial.

Reply:
Can I only add to  on<SignOutRequestEvent> in AuthBloc  aftre this emit(
        state.copyWith(authStatus: AuthStatus.unauthenticated, user: null),
      );  ?